product/models.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+from user.models import User
+
+class Brand(models.Model):
+    name         = models.CharField(max_length = 50)
+    small_image  = models.CharField(max_length = 2000)
+
+    class Meta:
+        db_table = 'brands'
+
+class FirstCategory(models.Model):
+    name    = models.CharField(max_length = 50)
+    brand   = models.ForeignKey('Brand', on_delete=models.SET_NULL, null = True)
+
+    class Meta:
+        db_table = 'firstcategories'
```

Comment:
테이블은 언더스코어로 의미 분리해주면 가독성이 좋아집니다.
```suggestion
        db_table = 'first_categories'
```